### PR TITLE
Added Appsembler Reporting hooks to edx-platform Ginkgo

### PR DESCRIPTION
**Pull request by johnbaldwin**
* removed the Ficus hooks in lms/envs/appsembler.py
* Added simple feature and INSTALLED_APP check in each of the devstack
and prod appsembler specific settings files

What's new is that we are also checking that 'appsembler_reporting' is
in the INSTALLED_APPS as well as if 'ENABLE_APPSEMBLER_REPORTING' is in
the server vars/lms.env.json 'FEATURES' section

# NOTE:
We may have to make additional changes, but so far this looks like this is all we need

### CODE REVIEW

**Comment by johnbaldwin** [comment]
You'll note that the previous feature is 'edx-figures' So I'll fix that and post another PR specific to Figures

**Comment by johnbaldwin** [comment]
[(5) Update and PR edx-platform Ginkgo for Appsembler Reporting support](https://trello.com/c/OeNmrJ93/502-5-update-and-pr-edx-platform-ginkgo-for-appsembler-reporting-support)

**Comment by OmarIthawi** [comment]
@johnbaldwin the tests are broken. Please take a look.

The tests were OK for https://github.com/appsembler/edx-platform/pull/259, so I think somethings wrong with this branch.

File: lms/envs/appsembler.py [hunk 1]
```diff
@@ -17,10 +17,3 @@
 INTERCOM_APP_ID = APPSEMBLER_FEATURES.get('INTERCOM_APP_ID', os.environ.get('INTERCOM_APP_ID', ''))
 INTERCOM_API_KEY = APPSEMBLER_FEATURES.get('INTERCOM_API_KEY', os.environ.get('INTERCOM_API_KEY', ''))
 INTERCOM_USER_EMAIL = APPSEMBLER_FEATURES.get('INTERCOM_USER_EMAIL', os.environ.get('INTERCOM_USER_EMAIL', ''))
-
-
-if APPSEMBLER_FEATURES.get('ENABLE_APPSEMBLER_REPORTING', False):
-    from appsembler_reporting.settings import APPSEMBLER_REPORTING
-
-    APPSEMBLER_REPORTING.update(APPSEMBLER_FEATURES.get(
-        'APPSEMBLER_REPORTING', {}))
```
Comment (OmarIthawi) [inline lms/envs/appsembler.py]: Can we still update `APPSEMBLER_FEATURES`, through `APPSEMBLER_FEATURES`?

Comment (johnbaldwin) [reply lms/envs/appsembler.py]: Good point. I was going to shift away from 'APPSEMBLER_FEATURES' and make it a top level setting in server-vars/lms.env.json settings. But reflecting on this, if we want common configuration behavior for Appsembler Reporting in Ficus and Ginkgo, then we'll either have to change how Ficus is configured (to be like the new Ginkgo config) or keep with 'APPSEMBLER_FEATURES'.

If we do go with a common ficus/ginkgo appsembler reporting release, then keeping with 'APPSEMBLER_FEATURES' means less changes for existing deployments, so I'll put the code back in to `appsembler.py`

Thanks for asking

**Comment by johnbaldwin** [comment]
Considering a number of factors, we're not updating edx-platform Ginkngo.

A) Appsembler reporting hooks are already in Ginkgo in `lms/envs/appsembler.py` This is a carry over from Ficus
B) Based on Omar's comment, I'm not going to change how server-vars need to be set up from Ficus to Ginkgo
C) I want Appsembler Reporting to have a common branch and release for Ficus and Ginkgo

**Pull request closed without merging**